specification/appendix-d-observability.md

```diff
@@ -67,7 +67,7 @@ The following table describes the history of changes to the OpenTelemetry featur
 ## Footnotes
 
 [^1]: The `variant` field should be included whenever possible as it represents the symbolic name of the flag's returned value (e.g., "on"/"off", "control"/"treatment"). Only omit if the provider doesn't supply this information.
-[^2]: The `value` field should be included whenever a `variant` is unavailable. Large or sensitive values should be redacted or omitted prior to being captured in telemetry signals.
+[^2]: The `value` field is required when a `variant` is not available, and recommended when it is. Considerations should be made for large and/or sensitive values, which should be redacted or omitted prior to being captured in telemetry signals.
 [^3]: Include `error.type` and `error.message`, if and only if an error occurred during a flag evaluation.
 
-[otel-ff-logs]: https://opentelemetry.io/docs/specs/semconv/feature-flags/feature-flags-logs/
+[otel-ff-events]: https://opentelemetry.io/docs/specs/semconv/feature-flags/feature-flags-logs/
```

Comment:
Does this link need to change later?

Reply:
@dyladan added an alias for us. This link should continue to work after the new semcon has been released. Good catch @askpt and thanks @dyladan.

https://github.com/open-telemetry/semantic-conventions/pull/2803